polkadot-parachains/canvas-kusama/src/xcm_config.rs

```diff
@@ -170,7 +176,7 @@ impl pallet_xcm::Config for Runtime {
 	type XcmExecuteFilter = Nothing;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 	type XcmTeleportFilter = Everything;
-	type XcmReserveTransferFilter = Everything;
+	type XcmReserveTransferFilter = Nothing;
```

Comment:
Okay so confirm the intention of this change, this will mean that anybody will still be able to `Teleport` any asset in and out of Canvas, but will **not** be able to perform `RevervedTransfer`s, correct?